src/devPropertiesWithoutUnitsRegExp.js

```diff
@@ -0,0 +1,19 @@
+let propertiesWitouthUnits
```

Comment:
Typo :)

Reply:
Haha. It was originally `propertiesWithUnits`, I was just off by one when I renamed it